var/spack/repos/builtin/packages/genemark-et/package.py

```diff
@@ -21,6 +21,7 @@ class GenemarkEt(Package):
     homepage = "http://topaz.gatech.edu/GeneMark"
     manual_download = True
 
+    version("4.71", sha256="629f430e7262bdb5df8f24413e65d26e35eb10ea34212145b692ee4689591e54")
```

Comment:
I verified this same checksum for the version that is currently listed as 4.72 on http://topaz.gatech.edu/GeneMark/license_download.cgi ... Should the version be updated to this?
```suggestion
    version("4.72", sha256="629f430e7262bdb5df8f24413e65d26e35eb10ea34212145b692ee4689591e54")
```

What I am missing is some info on which file this refers to. Should there be some info in the package description that this file is the one for "LINUX 64 kernel 3.10 - 5"?